tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+  "compilerOptions": {
+    "target": "ES2021",
+    "module": "commonjs",
```

Comment:
Any particular reason for `commonjs` versus an ES module here?

Reply:
```commonjs``` is just my default go to when I use typescript, so it was like muscle memory just not with any muscle. I have no problem changing it

Reply:
so changing it to esmodule caused some issues with index.ts so I think I will leave it as commonjs at least for now